packages/core/src/WebPlayer.ts

```diff
@@ -143,6 +143,35 @@ export default class WebPlayer extends EventEmitter {
     }
   }
 
+  changeVolume(change) {
```

Comment:
The common convention is to name things `set` not `change`, so `setVolume()`

Why do we have two methods for this?

Reply:
Addressed in new commit: Now using optional parameters in setVolume